web/src/Kustomization.jsx

```diff
@@ -57,7 +57,17 @@ export function Kustomization(props) {
         <span className='font-mono rounded text-neutral-600 bg-gray-100 px-1'>{item.spec.path}</span>
         }
       </div>
-      <div className="grid-cols-1 text-right">
+      <div className="grid grid-cols-1 text-right space-y-1">
+        <button className="bg-transparent hover:bg-neutral-100 font-medium text-sm text-neutral-700 py-1 px-2 border border-neutral-300 rounded"
+          onClick={() => capacitorClient.suspend("kustomization", item.metadata.namespace, item.metadata.name)}
+        >
+          Suspend
+        </button>
+        <button className="bg-transparent hover:bg-neutral-100 font-medium text-sm text-neutral-700 py-1 px-2 border border-neutral-300 rounded"
+          onClick={() => capacitorClient.resume("kustomization", item.metadata.namespace, item.metadata.name)}
+        >
+          Resume
+        </button>
```

Comment:
Maybe just add one button depending on the state of the object?  
If resource is already suspended, show resume
If resource is not suspended, show suspend

Something like this:
```
      <div className="grid-cols-1 text-right">
        <button className="bg-transparent hover:bg-neutral-100 font-medium text-sm text-neutral-700 py-1 px-2 border border-neutral-300 rounded"
          onClick={() => {
            if (source.spec.suspend) {
              capacitorClient.resume("kustomization", source.metadata.namespace, source.metadata.name);
            } else {
              capacitorClient.suspend("kustomization", source.metadata.namespace, source.metadata.name);
            }
          }}
        >
          {source.spec.suspend ? "Resume" : "Suspend"}
        </button>
      </div>
```

Of course would be needed for the other resource types as well.

Reply:
<img width="1512" alt="Screenshot 2024-04-08 at 19 01 45" src="https://github.com/gimlet-io/capacitor/assets/4289031/c9e8f35b-4415-4e35-965c-1dde6a5a60f1">